R-package/tests/testthat/test_utils.R

```diff
@@ -67,6 +67,7 @@ test_that("lgb.last_error() throws an error if there are no errors", {
 })
 
 test_that("lgb.last_error() correctly returns errors from the C++ side", {
+    testthat::skip("Skipping this test because it causes valgrind to think there is a memory leak, and needs to be rethought")
```

Comment:
Fix lint error:
```
/home/travis/build/microsoft/LightGBM/R-package/tests/testthat/test_utils.R:70:1: style: Lines should not be more than 120 characters.
    testthat::skip("Skipping this test because it causes valgrind to think there is a memory leak, and needs to be rethought")
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
fixed in https://github.com/microsoft/LightGBM/pull/3443/commits/0374e879821da2f2f85d72874092b28003632fc9